SIPS/sip-7.md

```diff
@@ -84,13 +92,14 @@ interface HeadingComponent extends BaseComponent {
 
 interface TextComponent extends BaseComponent {
   type: 'text';
-  text: string;
+  value: string;
 }
 
 interface ButtonComponent extends BaseComponent {
   type: 'button';
   variant?: 'primary' | 'secondary';
-  text: string;
+  buttonType?: 'button' | 'submit';
```

Comment:
```suggestion
  type?: 'button' | 'submit';
```

Reply:
we cannot use `type`. It's already used to determine what component is rendered